### PR TITLE
cryptocom add futures api

### PR DESCRIPTION
**Pull request by frosty00**
i need some apiKey from a verified user outside of these restricted countries:

https://help.crypto.com/en/articles/4894470-derivatives-trading-geo-restrictions

so if you can download the app get verified then send me ur apiKeys I can finish the private api for the derivates

also there's a bugfix in the orderbooks in this pr

### CODE REVIEW

**Comment by kroitor** [comment]
> so if you can download the app get verified then send me ur apiKeys I can finish the private api for the derivates

I'm in the verification process, will send you the keys when i have them.

File: js/cryptocom.js [hunk 1]
```diff
@@ -71,65 +71,105 @@ module.exports = class cryptocom extends Exchange {
...
+                        'get': {
+                            'public/auth': 1,
+                            'public/get-instruments': 1,
+                            'public/get-book': 1,
+                            'public/get-candlestick': 1,
+                            'public/get-ticker': 1,
+                            'public/get-trades': 1,
+                            'public/margin/get-transfer-currencies': 1,
+                            'public/margin/get-load-currenices': 1,
+                            'public/respond-heartbeat': 1,
```
Comment (kroitor) [inline js/cryptocom.js:95]: We could probably remove the `public/` and `private/` prefixes from these urls, cause the implicit methods get names like spotPublicGetPublicGetTicker, too much of PublicGet, no?

Comment (frosty00) [reply js/cryptocom.js:95]: i don't thin

> > so if you can download the app get verified then send me ur apiKeys I can finish the private api for the derivates
> 
> I'm in the verification process, will send you the keys when i have them.

please update me when you get it

**Comment by frosty00** [comment]
> > so if you can download the app get verified then send me ur apiKeys I can finish the private api for the derivates
> 
> I'm in the verification process, will send you the keys when i have them.

update me once you get it. I also recommend using the app for faster verification

File: js/cryptocom.js (hunk 1 above)
Comment (frosty00) [reply js/cryptocom.js:95]: if we remove them we'll need to have more string concatenation

Comment (kroitor) [reply js/cryptocom.js:95]: Yes, but we do the same everywhere anyway. Up to you.

**Pull request merged**